ServerCore/Pages/_Layout.cshtml

```diff
@@ -15,8 +15,16 @@
               asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
         <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
     </environment>
+
+    <!-- TODO: Remove when menus are implemented -->
+    <style>
+        .nav i { color: gray; font-size: 10px; }
+    </style>
 </head>
 <body>
+    @{ 
+        var Event = ViewData["Event"] as ServerCore.DataModel.Event;
```

Comment:
Is it necessary to use ViewData in layouts or is there some way to reference the model?

Reply:
The book I have says ViewData is needed for things in the shared file. With more engineering I am sure there is something more strongly typed but I am not sure it is worth it. I do have plans to automatically fill in the Event via routing and other things, which will help somewhat.